metagraph/src/graph/representation/hash/dbg_sshash.cpp

```diff
@@ -0,0 +1,234 @@
+#include "dbg_sshash.hpp"
+#include <dictionary.hpp>
```

Comment:
please update to the latest commit of sshash. I fixed a graph traversal bug that was affecting how we traverse our graphs.

Reply:
This has been fixed